_component_design/tabset.md

```diff
@@ -105,22 +170,38 @@ Pad the top and bottom of the content area by 24px. Pad the sides of the content
 
 <section class="static-section" markdown="1">
 
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
 ## States
 
+Tabs can exist in hover, pressed, or zero states.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
 <div class="hxRow"  markdown="1">
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
 ### Hover state
 
-Hovering over another tab changes the color of the text. Hovering off the tab changes it back to the default color.
+Construct the tab so that the text color changes to hex color #16b9d4 when a user hovers over the tab.
```

Comment:
When the cursor moves over any part of an unselected tab, change to the hover state. when the cursor moves off of the tab, the text should change back to the default color.